examples/testnet/Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: clean coldstart mrproper up
+all: up
+
+lisk_net = test
+lisk_database = lisk_$(lisk_net)
+
+blockchain.db.gz:
+	curl -O https://downloads.lisk.io/lisk/$(lisk_net)/blockchain.db.gz
+
+up:
+	docker-compose up -d
+
+coldstart: blockchain.db.gz up
+	docker-compose stop lisk
+	docker-compose start db
+	docker-compose run --rm task dropdb --if-exists $(lisk_database)
+	docker-compose run --rm task createdb $(lisk_database)
+	gzip --decompress --to-stdout blockchain.db.gz |docker-compose run --rm task psql >/dev/null
```

Comment:
+1 to give some output to the user. A progress bar or simply a sentence telling why we are waiting for.

Reply:
I considered adding a progress bar with `pv` but that would mean adding a dependency so I decided against it. Do you think it's really a must?

Reply:
If you use a sentence instead it is enough. I agree to not introduce more dependencies.